test/embedded/function_test.dart

```diff
@@ -1004,13 +1004,13 @@ void main() {
             });
 
             test("360", () async {
-              expect(
-                  await _deprotofy(_hsl(360, 50, 50, 1.0)), "hsl(0, 50%, 50%)");
+              expect(await _deprotofy(_hsl(360, 50, 50, 1.0)),
+                  "hsl(360, 50%, 50%)");
```

Comment:
We should still be normalizing polar angles to [0, 360).

Reply:
Updated in bbccca47560c19647c2c1842b52287e96f0699e9